_locales/es/messages.json

```diff
@@ -0,0 +1,16 @@
+{
+    "extensionName": {
+        "message": "Haz más preciso el texto \"No soy roboto\" de ReCaptcha",
```

Comment:
This looks like the same text as `extensionDescription`. They _are_ different, though only by one word... do the originals just translate to the exact same text?

Reply:
They are not identical in either language, so I have corrected the extension name. Thank you for pointing out the difference!

Reply:
Awesome, thanks very much for the translation! LGTM :thumbsup: